ddtrace/span.py

```diff
@@ -204,6 +204,11 @@ def set_tag(self, key, value=None):
         elif key == MANUAL_DROP_KEY:
             self.context.sampling_priority = priority.USER_REJECT
             return
+        elif key == SPAN_MEASURED_KEY:
+            # DEV: Force a consistent value of `1`
+            #      This also allows us to do `span.set_tag(SPAN_MEASURED_KEY)`
+            self.metrics[SPAN_MEASURED_KEY] = 1
```

Comment:
why is this using the `set_tag` interface while it seems to be a metric thing?
Why not expose the feature with something such as `set_measured()`?

Reply:
Not sure I want to own that public API just yet. We can always add that later.

As well, for anyone using OpenTracing (or in the future OpenTelemetry) interfaces for our tracers they won't have that, so will always need to make this work via `set_tag()`.